Kconfig.zephyr

```diff
@@ -248,6 +248,27 @@ config COMPILER_OPT
 
 endmenu
 
+choice
+	prompt "Error checking behavior"
+	default RUNTIME_ERROR_CHECKS
+
+config ASSERT_ON_ERRORS
+	bool "Assert on all errors"
+	help
+	  Assert on errors covered with the CHECK macro.
+
+config NO_RUNTIME_CHECKS
+	bool "No runtime error checks"
+	help
+	  Do not do any runtime checks or asserts.
```

Comment:
Nit: How it is phrased now, it seems to imply this option disables all asserts and error checking everywhere.

Would it be better to mention in the prompt above "Error checking behavior for CHECK macro" (and remove "covered with the CHECK macro" from the first option).
OR
repeat "covered with the CHECK macro" in the 2 others?

Reply:
sure.